lib/utilities/firebase_service.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class FirebaseService {
+  final FirebaseFirestore _firestore = FirebaseFirestore.instance;
```

Comment:
Keep in mind the dependency injection pattern. You should had inject this service.